rest-api-spec/src/main/resources/rest-api-spec/api/indices.rollover.json

```diff
@@ -0,0 +1,30 @@
+{
+  "indices.rollover": {
+    "documentation": "http://www.elastic.co/guide/en/elasticsearch/reference/master/indices-rollover-index.html",
+    "methods": ["POST"],
+    "url": {
+      "path": "/{index}/_rollover",
```

Comment:
Now we allow users to specify the new index name in the rollover request. If not specified we require the old index to have `${some_name}-${some_number}` format.